build.sbt

```diff
@@ -400,6 +402,7 @@ lazy val compiler = configureAsSubproject(project)
   .settings(generatePropertiesFileSettings)
   .settings(generateBuildCharacterFileSettings)
   .settings(Osgi.settings)
+  .settings(AutomaticModuleName.settings("scala.tools.nsc"))
```

Comment:
I wasn't to share what to call this one, or if we should include the attribute at all to the current, monolithic scala-compiler.jar

Reply:
@ktoso what do you think?

Reply:
To elaborate, I thought "scala.compiler" was a bit too broad for something that is, in historical context, just one of the succession of Scala compilers, each of which ends up in different packages.

Reply:
If you think this name is good then ok. Note that there's no rule to stick to package names really, but this sounds probably well namespaced 👍